### DISCUSSION TITLE
Trivia Bot

### DISCUSSION THREAD

**Discussion by BenLloydPearson** [mattermost/community#14] [Fun Ideas]
I would love to have a trivia bot that we can use as an incentive to get community participation on our chat server during major events. Ideally it would include features to:
* Preload multiple series of questions and answers 
* Schedule to run the trivia competitions at specific times.
* Use (hidden?) emojis for participants to select multiple-choice answers
* A leaderboard that shows who has answered the most questions correctly.

**Comment by larkox** [comment]
Really nice! I have already given two tries at this:
https://github.com/larkox/mattermost-plugin-colleaguequiz
https://github.com/larkox/mattermost-plugin-quiz
So that could be used as starting point :smile:
